Production_scripts/Process_extracts/extract_care_home.R

```diff
@@ -45,12 +45,6 @@ source_ch_clean <- matched_data %>%
   # compute lca variable from sending_location
   mutate(lca = convert_sending_location_to_lca(sending_location)) %>%
   # bed days
-  # create dummy end where blank
-  mutate(dummy_discharge = if_else(
-    is.na(record_keydate2),
-    end_fy(year) + days(1),
-    record_keydate2
-  )) %>%
   create_monthly_beddays(year, record_keydate1, dummy_discharge) %>%
```

Comment:
I think we need this here for calculating monthly beddays on line 54. But we could also add this into the function instead?

Reply:
Didn't realise this was already updated in calculate monthly beddays - This is fine